src/components/Post/CommentBox.jsx

```diff
@@ -60,7 +61,7 @@ const CommentBox = ({
 
           <IconButton
             className="post__button"
-            disabled={!comment}
+            disabled={!comment || comment.trim().length < 1}
```

Comment:
won't `comment.trim().length < 1` work the same as `!comment`? I think we wont need that

Reply:
`!comment` checks if its not null, undefined. But `comment.trim().length < 1` check if the comment is empty or contains only white spaces. 

But you are right. In this LOC, `!comment` seems redundant.